run_tests_with_cloudxr.sh fails when there is an active CloudXR runtime

**Describe the bug**
A clear and concise description of what the bug is.

**To reproduce**
1. Build an install Isaac Teleop

```
cmake -B build -DISAAC_TELEOP_PYTHON_VERSION=3.12
cmake --build build --parallel
cmake --install build

pip install isaacteleop[retargeters,cloudxr,ui] --find-links=./install/wheels/ --no-index --force-reinstall
```

2. Run CloudXR runtime

```
python -m isaacteleop.cloudxr
```

3. Run tests

```
./scripts/run_tests_with_cloudxr.sh --python-version 3.12
```

**Expected behavior**

Test should execute.

**Observed behavior**

```
[+] up 3/3
 ✔ Image isaacteleop-test-cloudxr-runtime Built                                                                                                                                   19.8s
 ✔ Volume isaacteleop-test_openxr-volume  Created                                                                                                                                 0.0s
 ✔ Container cloudxr-runtime-6.1.0-3.12   Started                                                                                                                                 21.9s
[INFO] Waiting for CloudXR runtime to be healthy...
..............................
[ERROR] CloudXR runtime failed to become healthy within 60s
[INFO] Container logs:
cloudxr-runtime-6.1.0-3.12  | INFO:              Vulkan Loader Version 1.3.275
cloudxr-runtime-6.1.0-3.12  | Running Isaac Teleop 1.1+local, CloudXR Runtime 6.1.0
cloudxr-runtime-6.1.0-3.12  | CloudXR runtime failed to start, terminating...
[INFO] Cleaning up containers...
[SUCCESS] Cleanup complete
```

**Platforms**
 - Isaac Teleop Version: [e.g. 1.0.23]
 - OS: [e.g. Ubuntu X86, ARM64]
 - Integrations: [e.g. Isaac Lab, Isaac Sim, ROS2]
 - XR Device: [e.g. Quest 3, Pico 4 Ultra]
 - Additional Devices [e.g. Manus]

**Additional context**
Add any other context about the problem here.

--- a/deps/cloudxr/docker-compose.runtime.yaml
+++ b/deps/cloudxr/docker-compose.runtime.yaml
@@ -16,7 +16,7 @@ services:
         ISAACTELEOP_PIP_DEBUG:
     container_name: cloudxr-runtime-${CXR_RUNTIME_SDK_VERSION:-unset}-${PYTHON_VERSION:-unset}
     user: "${CXR_UID:-1000}:${CXR_GID:-1000}"
-    network_mode: host
+    network_mode: "${CXR_RUNTIME_NETWORK_MODE:-host}"
     healthcheck:
       test: ["CMD", "test", "-f", "/openxr/.cloudxr/run/runtime_started"]
       interval: 1s

--- a/scripts/run_tests_with_cloudxr.sh
+++ b/scripts/run_tests_with_cloudxr.sh
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-# SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-FileCopyrightText: Copyright (c) 2025-2026 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
 # Test Runner Script with CloudXR
@@ -196,6 +196,9 @@ export CXR_PYTHON_GPU_TESTS="$CXR_PYTHON_GPU_TESTS_ENV"
 
 CXR_NATIVE_GPU_TESTS_ENV=$(IFS=','; echo "${CXR_NATIVE_GPU_TESTS[*]}")
 export CXR_NATIVE_GPU_TESTS="$CXR_NATIVE_GPU_TESTS_ENV"
+
+# Set CloudXR runtime network mode
+export CXR_RUNTIME_NETWORK_MODE="bridge"
 
 # Create/update .env file with test configuration
 log_info "Writing test configuration to $ENV_TEST..."

